src/test/java/racingcar/CarTest.java

```diff
@@ -26,18 +27,35 @@ class CarTest {
     @Order(2)
     @CsvSource({"4,9"})
     void 경주중_자동차가_이동하는_경우(int condition) {
-        Car car = new Car();
-        car.addMoveCount(condition);
-        assertThat(car.moveCountExpression("-")).isEqualTo("-");
+        Car car = new Car("test");
+        car.move(condition);
+        assertThat(car.checkPosition(1)).isTrue();
     }
 
     @ParameterizedTest
     @Order(3)
     @ValueSource(ints = {0, 3})
     void 경주중_자동차가_멈춰있는_경우(int condition) {
-        Car car = new Car();
-        car.addMoveCount(condition);
-        assertThat(car.moveCountExpression("-")).isBlank();
+        Car car = new Car("test");
+        car.move(condition);
+        assertThat(car.checkPosition(1)).isFalse();
```

Comment:
```suggestion
        assertThat(car.checkPosition(0)).isTrue();
```
멈춰있는 경우를 이렇게 표현해보면 어떨까요? :)

Reply:
네 변경 해보겠습니다!